src/apps/components/itemSheetsHeader/ItemSheetHeader.svelte

```diff
@@ -1,69 +1,118 @@
 <script>
-    import { getContext } from "svelte";
+  import { getContext } from "svelte";
 
-    import editDocumentImage from "../../handlers/editDocumentImage";
-    import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
+  import editDocumentImage from "../../handlers/editDocumentImage";
+  import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
 
-    const item = getContext("item");
+  const item = getContext("item");
+  const hasType = ["maneuver", "feature"];
```

Comment:
Should probably be renamed to better elaborate what it holds. Like `prerequisiteTypes`

Reply:
Agreed. Any variable name with a `has` or `is` should be a Boolean. I'd be very surprised to use this and find it's an array.